src/Employer/Employer.Web/Controllers/ApplicationReviewController.cs

```diff
@@ -18,25 +18,47 @@ public ApplicationReviewController(ApplicationReviewOrchestrator orchestrator)
             _orchestrator = orchestrator;
         }
 
-        [HttpGet("", Name = RouteNames.ApplicationReview_Get)]
+        [HttpGet("review", Name = RouteNames.ApplicationReview_Get)]
         public async Task<IActionResult> ApplicationReview(ApplicationReviewRouteModel rm)
         {
             var vm = await _orchestrator.GetApplicationReviewViewModelAsync(rm);
             return View(vm);
         }
 
-        [HttpPost("", Name = RouteNames.ApplicationReview_Post)]
+        [HttpPost("review", Name = RouteNames.ApplicationReview_Post)]
         public async Task<IActionResult> ApplicationReview(ApplicationReviewEditModel m)
         {
-            if (ModelState.IsValid == false)
+            if (!ModelState.IsValid)
             {
                 var vm = await _orchestrator.GetApplicationReviewViewModelAsync(m);
                 return View(vm);
-            }
+            }            
+            var applicationStatusConfirmationViewModel = await _orchestrator.GetApplicationStatusConfirmationViewModelAsync(m);
+            return View("ApplicationStatusConfirmation", applicationStatusConfirmationViewModel);
+        }
 
-            await _orchestrator.PostApplicationReviewEditModelAsync(m, User.ToVacancyUser());
+       [HttpPost("status", Name = RouteNames.ApplicationStatusConfirmation_Post)]
+        public async Task<IActionResult> ApplicationStatusConfirmation(ApplicationReviewStatusConfirmationEditModel applicationReviewStatusConfirmationEditModel)
+        {            
+            if (!ModelState.IsValid)
+            {
+                var vm = await _orchestrator.GetApplicationStatusConfirmationViewModelAsync(applicationReviewStatusConfirmationEditModel);
+                return View(vm);
+            }
 
-            return RedirectToRoute(RouteNames.VacancyManage_Get);
-        }
+            if (applicationReviewStatusConfirmationEditModel.NotifyApplicant.HasValue &&
```

Comment:
Can you create a helper property off of the model for this. `IsNotifyingApplication` or something. At first glance I'm not sure what this `if` condition means.